targets/targets.json

```diff
@@ -2442,19 +2459,18 @@
         "public": false,
         "detect_code": ["1101"],
         "program_cycle_s": 6,
-        "MERGE_SOFT_DEVICE": true,
         "EXPECTED_SOFTDEVICES_WITH_OFFSETS": [
             {
-                "boot": "",
+                "boot": ["s132_nrf52_2.0.0_db_bootloader.hex", "s132_nrf52_2.0.0_sb_bootloader.hex"],
```

Comment:
There is a typo in the dual bank bootloader name: `s132_nrf52_2.0.0_db_bootloadre.hex` instead of `s132_nrf52_2.0.0_db_bootloader.hex`